mkdocs/config/base.py

```diff
@@ -21,13 +21,14 @@ class Config(utils.UserDict):
     for running validation on the structure and contents.
     """
 
-    def __init__(self, schema):
+    def __init__(self, schema, fname=None):
         """
         The schema is a Python dict which maps the config name to a validator.
         """
 
         self._schema = schema
         self._schema_keys = set(dict(schema).keys())
+        self.fname = fname
```

Comment:
I'm not crazy about naming this `fname`. Considering its intended use, we don't care about the "name" of the file, but rather the "path" to the file. Using `fname` indicates to me that I only need to pass in the name of the file, not the full path. However, in actual use, I need to pass in the absolute path (and it might even work if I left off the filename). Therefore, let's indicate that with the name used.

Reply:
Maybe `config_path`?